app/views/auth.html

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
What is the use of this template? :sweat_smile:

Reply:
@Autophagy authentication directive requires a template.. 🤔i need to find a better solution, for now i replaced with an inline template